src/NumericalAlgorithms/DiscontinuousGalerkin/NumericalFluxes/Hll.hpp

```diff
@@ -173,6 +173,13 @@ struct Hll {
         get(max_signal_speed)[s] = std::max(get(max_signal_speed_interior)[s],
                                             -get(min_signal_speed_exterior)[s]);
       }
+      ASSERT(min(get(max_signal_speed) - get(min_signal_speed)) > 0.,
```

Comment:
[nitpick] `0.0` since there's `1.0` below

Reply:
Oh no! Thanks for catching :)

Reply:
Done and squashed